eng/scripts/DevOps-Recordings-Download.ps1

```diff
@@ -0,0 +1,62 @@
+param ($pr, $noWait = $false)
```

Comment:
Nit: PowerShell recommends scripts follow cmdlet rules, i.e. <approved verb>-<prefix><noun>, so maybe Download-DevOpsRecordings.ps1 (and something simlar for the other).

Also, I redcommend attributing this with `[CmdletBinding()]` (parens are necessary in PowerShell). It actually provides more built-in factilities, like passing down whether `-verbose` was passed via script-scopped `$VerbosePreference` getting set. You should also declare types on the params, though the `[Parameter()]` attribute isn't really necessary. E.g.

```powershell
[CmdletBinding()]
param (
  [int] $pr,
  [switch] $NoWait = $false
)
```

This helps people know what theys should even pass, and pushes type validation to PowerShell rather than you dealing with it.